node/silkworm/etl/buffer.cpp

```diff
@@ -20,30 +20,5 @@
 
 namespace silkworm::etl {
```

Comment:
I believe buffer.cpp can be safely removed.

Reply:
yes, will do